Added Changelog for Feb and March (#5653)
codesandbox/codesandbox-client: packages/homepage/content/changelog/4-0-2/index.md

---
banner: ./4-0-2.jpg
title: v4.0.2
slug: Custom Artwork
date: 2021-03-21

> description: Likes and shared with you, now in the dashboard and Publish to GitHub Pages

Reviewer: ```suggestion
description: Likes and Shared With You pages in the dashboard and the ability to publish to GitHub Pages
```

---

#### Features

> 1. Improve esmodule detection 

Reviewer: ```suggestion
1. Improve esmodule detection in the bundler
```

> 2. Preload babel.min.js from root 

Reviewer: ```suggestion
2. Improve initial loading speed of the bundler
```

> 3. Github pages deploy 

Reviewer: ```suggestion
3. Added support for deploying to GitHub Pages
```

> 4. Added Shared with me & Liked page 

Reviewer: ```suggestion
4. Added Shared With Me & Liked page in the dashboard. Now you can see which sandboxes have been shared with you and which ones you've liked before. 
```

> 5. Add Basic auth as option in private npm 

Reviewer: ```suggestion
5. Add Basic auth support in private npm, making it possible to use private npm with Azure DevOps and Artifactory
```
